src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -36,74 +38,92 @@ const propTypes = {
         super(props);
+        this.validate = this.validate.bind(this);
 
         this.addBeneficialOwner = this.addBeneficialOwner.bind(this);
         this.submit = this.submit.bind(this);
 
         this.state = {
             ownsMoreThan25Percent: ReimbursementAccountUtils.getDefaultStateForField(props, 'ownsMoreThan25Percent', false),
             hasOtherBeneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'hasOtherBeneficialOwners', false),
-            acceptTermsAndConditions: ReimbursementAccountUtils.getDefaultStateForField(props, 'acceptTermsAndConditions', false),
-            certifyTrueInformation: ReimbursementAccountUtils.getDefaultStateForField(props, 'certifyTrueInformation', false),
             beneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'beneficialOwners', []),
         };
-
-        // These fields need to be filled out in order to submit the form (doesn't include IdentityForm fields)
-        this.requiredFields = [
-            'acceptTermsAndConditions',
-            'certifyTrueInformation',
-        ];
-
-        // Map a field to the key of the error's translation
-        this.errorTranslationKeys = {
-            acceptTermsAndConditions: 'common.error.acceptedTerms',
-            certifyTrueInformation: 'beneficialOwnersStep.error.certify',
-        };
-
-        this.getErrors = () => ReimbursementAccountUtils.getErrors(this.props);
-        this.clearError = inputKey => ReimbursementAccountUtils.clearError(this.props, inputKey);
-        this.clearErrors = inputKeys => ReimbursementAccountUtils.clearErrors(this.props, inputKeys);
-        this.getErrorText = inputKey => ReimbursementAccountUtils.getErrorText(this.props, this.errorTranslationKeys, inputKey);
     }
 
     /**
-     * @returns {Boolean}
+     * @param {Object} values - input values passed by the Form component
+     * @returns {Object}
      */
-    validate() {
-        let beneficialOwnersErrors = [];
+    validate(values) {
+        const errors = {};
+
+        const errorKeys = {
+            street: 'address',
+            city: 'addressCity',
+            state: 'addressState',
+        };
+        const requiredFields = ['firstName', 'lastName', 'dob', 'ssnLast4', 'street', 'city', 'zipCode', 'state'];
         if (this.state.hasOtherBeneficialOwners) {
-            beneficialOwnersErrors = _.map(this.state.beneficialOwners, ValidationUtils.validateIdentity);
+            _.each(this.state.beneficialOwners, (ownerKey) => {
+                // eslint-disable-next-line rulesdir/prefer-early-return
+                _.each(requiredFields, (inputKey) => {
+                    if (!ValidationUtils.isRequiredFulfilled(values[`beneficialOwner.${ownerKey}.${inputKey}`])) {
+                        const errorKey = errorKeys[inputKey] || inputKey;
+                        errors[`beneficialOwner.${ownerKey}.${inputKey}`] = this.props.translate(`bankAccount.error.${errorKey}`);
```

Comment:
NAB, I don't know if this is already an established pattern, but I find it a a bit strange that we are using the `.` symbol in the object key. 

Mainly, when I first looked at this I got confused about whether it's dot notation e.g. the way someone might use [`lodash.get()` to access a property](https://lodash.com/docs/4.17.15#get) somewhere (maybe inside the `Form` component) or if we have an object with property names like this:

```
{
    'beneficialOwner.123456.firstName': 'Bob',
}
```

I'd probably suggest we use something else like an `_` to help avoid any confusion, but if we're already doing this all over the place maybe it is less important to solve in this PR.

Reply:
Oh sorry, can you elaborate on why the period before `firstName` is required?

Reply:
I meant to update, my bad- it's not required! I pushed that change already